website/content/maintainers/templates/_index.md

```diff
@@ -5,6 +5,8 @@ layout: single
 date: 2020-11-07
 description: >
   Jump start your CNCF project by using our templates
+aliases:
+- /maintainers/github/templates/_index
```

Comment:
In the case of the index pages, the alias should be as follows (below) because the URL of the page that we are redirecting from is https://contribute.cncf.io/maintainers/github/templates/


```suggestion
- /maintainers/github/templates/
```

Reply:
Thanks! I just pushed a commit with this change :)